packages/backend/src/createRedirect.ts

```diff
@@ -73,6 +73,7 @@ type CreateRedirect = <ReturnType>(params: {
 }) => {
   redirectToSignIn: RedirectFun<ReturnType>;
   redirectToSignUp: RedirectFun<ReturnType>;
+  redirectToTasks: RedirectFun<ReturnType>;
```

Comment:
Another option is to put the redirection logic within `redirectToSignIn` instead of introducing a separate `redirectToTasks`

This could work for the Astro case as well